frontend/src/store/index.js

```diff
@@ -284,6 +284,20 @@ export const store = new Vuex.Store({
       }
       return ''
     },
+
+    liaison_designate(state) {
+      if (state.user && state.user.role && state.user.role.role_code){
```

Comment:
This check doesn't need to check for `state.user.role`

Reply:
fixed